modules/migrations/migrate.go

```diff
@@ -63,6 +64,11 @@ func MigrateRepository(doer *models.User, ownerName string, opts base.MigrateOpt
 	}
 
 	uploader.gitServiceType = opts.GitServiceType
+
+	if setting.Migrations.RetryMaxTimes > 0 {
+		downloader = base.NewRetryDownloader(downloader, setting.Migrations.RetryMaxTimes+1, setting.Migrations.RetryDelay)
```

Comment:
This `RetryMaxTimes+1` is what made got me thinking about the meaning of `RETRY_MAX_TIMES`.